src/libs/auth.ts

```diff
@@ -370,8 +395,8 @@ const becameRegistered = (oldState: AuthState, state: AuthState) => {
   return oldState.registrationStatus !== 'registered' && state.registrationStatus === 'registered';
 };
 
-export const isAuthSettled = ({ isSignedIn, registrationStatus }) => {
-  return isSignedIn !== undefined && (!isSignedIn || registrationStatus !== undefined);
+export const isAuthSettled = (state: AuthState) => {
+  return (!state.isSigningIn && !state.isSignedIn) || state.registrationStatus !== 'uninitialized';
```

Comment:
It looks like the parentheses got rearranged here and thus the logic changed.

Reply:
I will try to be more careful next time